appveyor.yml

```diff
@@ -41,6 +41,6 @@ install:
  - cargo -V
 build: false
 test_script:
-    - cargo test --verbose --features appveyor
+    - cargo test --verbose --features 'appveyor serde'
```

Comment:
For Travis, you should just be able to add this to the features matrix, and the build script will pick it up:

https://github.com/libpnet/libpnet/blob/3f8d1ccfdb96b70c3fea07464582f24ccd66e178/.travis.yml#L15